src/app/core/layers/layersstore.js

```diff
@@ -255,8 +255,14 @@ proto.setLayersTree = function(layerstree, name, expanded=true) {
   };
   const traverse = (nodes, parentGroup) => {
     nodes.forEach((node, index) => {
-      // case of layer substitute node with layere state
-      if (node.id !== undefined) nodes[index] = this.getLayerById(node.id).getState();
+      // case of layer substitute node with layer state
+      if (node.id !== undefined) {
+        nodes[index] = this.getLayerById(node.id).getState();
+        /**
+         * @since 3.8.0
+         */
+        nodes[index].showfeaturecount = node.showfeaturecount || false;
```

Comment:
Not a big fan of this kind of solution, is it possible to change the `getstate()` function internally instead? (that is, returning also the `showfeaturecount` property?)